conf/base.config

```diff
@@ -83,12 +83,23 @@ process {
 
     // PYTHON BASED MODULES WHICH SHOULDN'T NEED MORE THAN 1 CORE AND A MIDDLING AMOUNT OF MEMORY
     // WILL BE REVIEWED
-    withName: 'VALIDATE_TAXID|TRAILINGNS|GC_CONTENT|GET_KMERS_PROFILE|PARSE_FCSGX_RESULT|ASCC_MERGE_TABLES|GET_LARGEST_SCAFF|KMER_COUNT_DIM_REDUCTION|KMER_COUNT_DIM_REDUCTION_COMBINE_CSV|REFORMAT_DIAMOND_OUTFMT6|CONVERT_TO_HITS_FILE|DIAMOND_BLAST_CHUNK_TO_FULL|GENERATE_SAMPLESHEET' {
+    withName: 'VALIDATE_TAXID|TRAILINGNS|GC_CONTENT|PARSE_FCSGX_RESULT|ASCC_MERGE_TABLES|GET_LARGEST_SCAFF|KMER_COUNT_DIM_REDUCTION|KMER_COUNT_DIM_REDUCTION_COMBINE_CSV|REFORMAT_DIAMOND_OUTFMT6|CONVERT_TO_HITS_FILE|DIAMOND_BLAST_CHUNK_TO_FULL|GENERATE_SAMPLESHEET' {
         cpus    = { 1                           }
         memory  = { 10.GB    * task.attempt     }
         time    = { 5.h      * task.attempt     }
     }
 
+    withName: "GET_KMERS_PROFILE" {
```

Comment:
Should this be GET_KMER_COUNTS - GET_KMERS_PROFILE is the name of the subworkflow

Reply:
yes, Done